.github/workflows/version.yml

```diff
@@ -5,8 +5,8 @@ name: Version
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
 on:
-  pull_request_target:
-
+  push:
+    branches: master
```

Comment:
You could have this run on tags, but then it can't update the tag, because the tag already exists. I am not sure what the best way out of that cycle is.